x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -259,39 +260,23 @@ export const getManagementFilteredLinks = async (
         )
       : getEndpointAuthzInitialState();
 
-    // exclude links based on canAccessEndpointManagement
-    if (!canAccessEndpointManagement) {
-      return excludeLinks([
-        SecurityPageName.hostIsolationExceptions,
-        SecurityPageName.responseActionsHistory,
-      ]);
+    if (!canReadActionsLogManagement) {
+      linksToExclude.push(SecurityPageName.responseActionsHistory);
     }
 
-    // exclude links based on privileges and HIE entries
-    if (!canReadActionsLogManagement) {
-      // <= enterprise license
+    if (!canIsolateHost && canUnIsolateHost) {
```

Comment:
here I used `canUnIsolateHost` next to the already used `canIsolateHost`, because it does not incorporate the license, while `canIsolateHost` does, see in authz.ts:
```
    canIsolateHost: canIsolateHost && isPlatinumPlusLicense,
    canUnIsolateHost: canIsolateHost,
```

Although, I think this is not final - rather `canReadHostIsolationExceptions` needs to be used.